docs/current/cookbook/snippets/cache-dependencies/index.mjs

```diff
@@ -0,0 +1,25 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(
+  async (client) => {
+    // use a node:18 container
+    // mount the source code directory on the host
+    // at /src in the container
+    // mount the cache volume to persist dependencies
+    const source = client
+      .container()
+      .from("node:18")
+      .withDirectory("/src", client.host().directory("."))
+      .withMountedCache("/src/node_modules", client.cacheVolume("node"))
```

Comment:
What happens if some other pipeline that uses the same name for the cache volume runs at the same time?

In the past - a few months ago - I used to have issues when multiple operations would access the same cache volume concurrently. I fixed it by scoping these cache volumes, e.g. `node-18-<APP>-<ENV>`.

Same applies to all other languages.

Reply:
Done